packages/interaction-viewer/src/AdjacencyGraph.ts

```diff
@@ -0,0 +1,303 @@
+/* eslint-disable no-param-reassign */
+import { html, TemplateResult } from "lit";
+import { select, mouse, scaleBand, scaleLinear } from "d3";
+import InteractionTooltip from "./interaction-tooltip";
+import { Disease, Interaction, SubcellularLocation } from "./data";
+import { ProcessedData } from "./apiLoader";
+
+const formatDiseaseInfo = (
+  disease: Disease,
+  accession: string
+): TemplateResult => {
+  if (disease.dbReference) {
+    // Some have only text
+    return html`<p>
+      <a
+        href="//www.uniprot.org/uniprot/${accession}#${disease.acronym}"
+        target="_blank"
+        >${disease.diseaseId}</a
+      >
+    </p>`;
+  }
+  return html`<p>${disease.diseaseId}</p>`;
+};
+
+const formatSubcellularLocationInfo = (
+  subcell: SubcellularLocation
+): TemplateResult =>
+  html`${subcell.locations.map(
+    (location) =>
+      html`<p>
+        ${location.location?.value}${location.topology
+          ? `(${location.topology.value})`
+          : ""}
+      </p>`
+  )}`;
+
+const drawAdjacencyGraph = (
+  containerElt: HTMLElement,
+  accession: string,
+  processedData: ProcessedData,
+  tooltip: InteractionTooltip,
+  interactionViewerElt: HTMLElement
+): void => {
+  const { adjacencyMap, entryStore, interactionsMap } = processedData;
+
+  const margin = {
+    top: 100,
+    right: 0,
+    bottom: 10,
+    left: 100,
+  };
+
+  const width = 18 * adjacencyMap.length;
+
+  const height = width;
+
+  const x = scaleBand().rangeRound([0, width]);
+  const intensity = scaleLinear().range([0.2, 1]);
+
+  const svg = select(containerElt)
+    .append("svg")
+    .attr("width", width + margin.left + margin.right)
+    .attr("height", height + margin.top + margin.bottom)
+    .attr("class", "interaction-viewer")
+    .append("g")
+    .attr("class", "interaction-viewer-group")
+    .attr("transform", `translate(${margin.left},${margin.top})`);
+
+  x.domain(adjacencyMap.map(({ accession }) => accession));
+  intensity.domain([0, 10]);
+
+  // x.domain(nodes.map(entry => entry.accession)); intensity.domain([0,
+  // d3.max(nodes.map(link => link.experiments))]);
+
+  const getInteractionData = (
+    accession1: string,
+    accession2: string
+  ): Interaction => {
+    let data = interactionsMap.get(`${accession1}${accession2}`);
+    if (!data) {
+      data = interactionsMap.get(`${accession2}${accession1}`);
+    }
+    if (!data) {
+      console.error(`Interaction not found for ${accession1}:${accession2}`);
+    }
+    return data;
+  };
+
+  const getIntactLink = (interactor1: string, interactor2: string): string => {
+    return `//www.ebi.ac.uk/intact/query/id:${interactor1} AND id:${interactor2}`;
+  };
+
+  const mouseover = (accession1: string, accession2: string) => {
+    // svg.classed("active-cell", true);
+
+    const group = svg.insert("g", ":first-child").attr("class", "active-group");
```

Comment:
```suggestion
    const group = svg.insert("g").attr("class", "active-group");
```
In order to have the line go over the grey triangle so that it's easier to see to what accession we are pointing